modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java

```diff
@@ -368,67 +444,217 @@ private void _testSystemObjectRelationshipManyToMany() throws Exception {
 				objectRelationshipTypeException.getMessage());
 		}
 
+		try {
+			_objectRelationshipLocalService.addObjectRelationship(
+				TestPropsValues.getUserId(),
+				_systemObjectDefinition1.getObjectDefinitionId(),
+				_objectDefinition1.getObjectDefinitionId(),
+				RandomTestUtil.randomLong(),
+				ObjectRelationshipConstants.DELETION_TYPE_PREVENT,
+				LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
+				StringUtil.randomId(),
+				ObjectRelationshipConstants.TYPE_MANY_TO_MANY);
+		}
+		catch (ObjectRelationshipParameterObjectFieldIdException
+					objectRelationshipParameterObjectFieldIdException) {
+
+			Assert.assertEquals(
+				"Object relationship type " +
+					ObjectRelationshipConstants.TYPE_MANY_TO_MANY +
+						" does not allow a parameter object field ID",
+				objectRelationshipParameterObjectFieldIdException.getMessage());
+		}
+
 		_objectRelationshipLocalService.addObjectRelationship(
 			TestPropsValues.getUserId(),
 			_objectDefinition1.getObjectDefinitionId(),
-			_systemObjectDefinition.getObjectDefinitionId(), 0,
+			_systemObjectDefinition2.getObjectDefinitionId(), 0,
 			ObjectRelationshipConstants.DELETION_TYPE_PREVENT,
 			LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
 			StringUtil.randomId(),
 			ObjectRelationshipConstants.TYPE_MANY_TO_MANY);
 	}
 
 	private void _testSystemObjectRelationshipOneToMany() throws Exception {
-		ObjectRelationship objectRelationship =
+		ObjectRelationship objectRelationship1 =
```

Comment:
Why did you rename it to `objectRelationship1`? I couldn't find the `objectRelationship2`

Reply:
You are right. I think I used to have one, but forgot to rename it after I removed it.